moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp

```diff
@@ -404,6 +399,10 @@ void RobotStateDisplay::onDisable()
 void RobotStateDisplay::update(float wall_dt, float ros_dt)
 {
   Display::update(wall_dt, ros_dt);
+
+  if (load_robot_model_)
+    loadRobotModel();
+
```

Comment:
@rhaschke do you think you'll look into making this a background job, or do you think this PR is enough for now in simply fixing the misleading error messages? i'll test and review if you consider it done